e2-studio-star-rx72n-firmware/src/main.c

```
@@ -1874,7 +1874,11 @@ void tx_application_define(void* first_unused_memory)
   RX_ASSERT(err == k_rx_ok, "led_status_task_create must succeed");
 
   /* BMS Monitor Task - Priority 15 */
-  err = bms_monitor_task_create();
+  const bms_monitor_config_t bms_config = {
```

Clang (14.0.6)

[warning] variable 'bms_config' is not initialized

(cppcoreguidelines-init-variables)